PixelHunter1995/Components/SpeechLine.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace PixelHunter1995.Components
+{
+    internal class SpeechLine : IUpdateable
+    {
+        private string ActiveSpeech;
```

Comment:
This name seems unclear. Maybe `SpeechText`, `SpeechContent`, `DialogText` or something?

Reply:
Maybe you want to keep the "Active" part, or use "Current".